contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -142,11 +137,12 @@
             right: 'initial',
           },
         },
-        activeImageComponent: null,
+        activeImageField: null,
         uploadingChecksum: '',
         mathQuill: null,
         keyDownEventListener: null,
         clickEventListener: null,
+        editImageEventListener: null,
```

Comment:
Instead of listening to `'edit'` events on individual `MarkdownImageField` components, we're now using a global listener instead, which simplifies cleanup and eliminates some async quirks.